_includes/scripts.html

```diff
@@ -1,5 +1,5 @@
 <!-- jQuery first, then Bootstrap JS. -->
-<script src="https://cdn.llnl.gov/jquery/2.1.4/js/jquery-2.1.4.min.js"></script>
+<script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
```

Comment:
Is there a specific feature that requires the newer version of jQuery? Have we reviewed the impact of the known breaking changes on the rest of the site? (https://jquery.com/upgrade-guide/)

We'll also want the new version added to our own CDN. I'll submit that update ASAP.

Reply:
jQuery is also a dependency of Bootstrap; We would need to verify that version compatibility as well, since our current version of Bootstrap is pre jQuery 3 and its breaking changes. (Bootstrap 3.3.5 released in 2015 [with jQuery requirements of only 1.9.1 and above](https://github.com/twbs/bootstrap/blob/v3.3.5/bower.json#L32), jQuery 3.0.0 released in 2016)

Reply:
I have not been able to implement event.preventDefault() in 2.1.4, which I need for the search feature.

Reply:
> I have not been able to implement event.preventDefault() in 2.1.4, which I need for the search feature.

I think it exists in 2.1.4, so I'm not sure why that would be. But it looks like the search currently isn't working either way....

Reply:
I'm wondering if it has something to do with this? https://javascript.developreference.com/article/20521561/PreventDefault+not+preventing+form+to+send
"You need to delay the addition of your event-listener function until the browser has finished loading the page. Otherwise those lines of JavaScript get executed before the form element exists, so the function doesn't get attached to it."

Reply:
Hmm, that's odd. I am getting that too now. I showed it to Ian last week and it worked. It also worked yesterday. Might have something to do with the most recent commits somehow.

Reply:
Yes, search worked [here](https://github.com/LLNL/llnl.github.io/pull/450/commits/f16a5bd632853f6346b69063a1d3521cc6417e3e)

Reply:
The problem with search seems to be the name actually.